code/game/objects/items/robot/items/storage.dm

```diff
@@ -300,3 +300,40 @@
 	if(istype(atom, /obj/item/ai_module) && !stored) //If an admin wants a borg to upload laws, who am I to stop them? Otherwise, we can hint that it fails
 		to_chat(user, span_warning("This circuit board doesn't seem to have standard robot apparatus pin holes. You're unable to pick it up."))
 	return ..()
+
+/obj/item/borg/apparatus/service
+	name = "Service apparatus"
+	desc = "A special apparatus for carrying food, bowls, plates, oven trays, soup pots and paper."
+	icon_state = "borg_service_apparatus"
+	storable = list(
+				/obj/item/reagent_containers/cup/bowl,\
+				/obj/item/food,\
+				/obj/item/plate/oven_tray,\
+				/obj/item/paper,\
+				/obj/item/plate,\
+				/obj/item/reagent_containers/cup/soup_pot,\
+)
```

Comment:
```suggestion
	storable = list(
		/obj/item/reagent_containers/cup/bowl,
		/obj/item/food,
		/obj/item/plate/oven_tray,
		/obj/item/paper,
		/obj/item/plate,
		/obj/item/reagent_containers/cup/soup_pot,
	)
```

each indented entry should only be one indent greater than the indent of the line the `list()` was started on. since line 308 has one indent, every entry of the list should have two indents. we also have the close parentheses at the same indent level of the line that the `list()` was started on.

(also ditto on the whole backslash thing)